### ISSUE TITLE
 Chronograf (1.8.0) Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by linucks** [influxdata/chronograf#5449]
What browser are you using (name and version)?
Brave Version 1.5.123 Chromium: 80.0.3987.163 (Official Build) (64-bit)

What operating system are you using?
Mac OSX 10.14.6

Please describe what you were trying to do when you encountered this error:
I'd just started Chronograf for the first time with:

chronograph --port 8889

and was looking around the interface. I was starting on port 8889 as 8888 conflicted with Visual Studio Code

```
TypeError: Cannot read property 'name' of undefined
    at http://localhost:8889/src.2ed66c05.js:2365:385
    at Array.map (<anonymous>)
    at l (http://localhost:8889/src.2ed66c05.js:2365:310)
    at sa (http://localhost:8889/src.2ed66c05.js:717:55869)
    at $a (http://localhost:8889/src.2ed66c05.js:717:68127)
    at oo (http://localhost:8889/src.2ed66c05.js:717:91055)
    at uo (http://localhost:8889/src.2ed66c05.js:717:91442)
    at Yo (http://localhost:8889/src.2ed66c05.js:717:98524)
    at $o (http://localhost:8889/src.2ed66c05.js:717:97904)
    at eu (http://localhost:8889/src.2ed66c05.js:717:99279)
```

**Comment by russorat** [comment]
@linucks thanks! i believe this was fixed with 1.8.2. Could you try upgrading?

**Comment by timhallinflux** [comment]
@linucks -- did the upgrade resolve your issue?  Given the age of this issue, I'd propose closing it if we don't hear back.
Thanks.
